app/presenters/check_answers/case_disposal_card.rb

```diff
@@ -0,0 +1,18 @@
+module CheckAnswers
+  class CaseDisposalCard < Base
+    attr_reader :case_disposal_form
+
+    KEY = 'case_disposal'.freeze
+    GROUP = 'about_case'.freeze
```

Comment:
Should we not be using the pragma `# frozen_string_literal: true` rather than freezing everything

Reply:
updated